Exiled.Events/EventArgs/Player/TogglingRadioEventArgs.cs

```diff
@@ -0,0 +1,67 @@
+// -----------------------------------------------------------------------
+// <copyright file="TogglingRadioEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Player
+{
+    using API.Features;
+
+    using Exiled.API.Features.Items;
+    using Interfaces;
+    using InventorySystem.Items.Radio;
+
+    /// <summary>
+    ///     Contains all information before toggling a radio.
```

Comment:
```suggestion
    /// Contains all information before toggling a radio.
```

Remove all white spaces as they're useless and it's horrible.
Do that for all occurrences.